recipes/libspatialite/all/conanfile.py

```diff
@@ -192,14 +198,17 @@ def _build_msvc(self):
         if not self.options.with_minizip:
             replace_in_file(self, gaiaconfig_msvc, "#define ENABLE_MINIZIP 1", "")
 
+        # Workaround for a NMakeDeps define quoting issue:
```

Comment:
```suggestion
        # Workaround for a NMakeDeps define quoting issue. Fixed in Conan 1.63 and 2.0.15
```

So we can remember in the future to be removed. It's still a fresh bugfix.